.travis.yml

```diff
@@ -132,3 +132,8 @@ script:
 
 notifications:
   email: false
+
+branches:
+  exclude:
+    - /^sils\//
+    
```

Comment:
just remove the spaces here, keep that newline ;) you can check on github if it's there, there's a red symbol if there's no newline before EOF